File: Data Structure using java/_M_Dynamic_Queue/src/a1_employee/Employee.java
package a1_employee;

public class Employee
{
	int e_id;
	String name;
	
	public Employee(int e_id, String name)
	{
		this.e_id = e_id;
		this.name = name;
	}

	@Override
	public String toString()
	{
		return "["+e_id+", "+name+"]";
	}
	
	
	
	
}

File: .net/DBFirst/Models/Employee.cs
using System;
using System.Collections.Generic;

namespace DBFirst.Models
{
    public partial class Employee
    {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public string Email { get; set; } = null!;
        public string Password { get; set; } = null!;
        public string Contact { get; set; } = null!;
        public string Address { get; set; } = null!;
        public int DepartmentId { get; set; }

        public virtual Department Department { get; set; } = null!;
    }
}
